src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -269,4 +273,27 @@ public static AssemblyResultSet assembleReads(final AssemblyRegion region,
             throw e;
         }
     }
+
+    /**
+     * @return the default set of read filters for HC (includes the MQ threshold filter) or M2 (does not include the MQ filter)
+     */
+    public static List<ReadFilter> makeStandardReadFilterList(final boolean includeMQThresholdFilter) {
```

Comment:
I'd rather not have the make read filters methods for Mutect and HaplotypeCaller entangled like this -- it is very problematic for adding additional filters in the future to one tool but not the other. Can you restore `HaplotyeeCallerEngine.makeStandardHCReadFilters()`, and give Mutect a completely separate method for creating its filters?